modules/react/button/stories/button/examples/CustomStyles.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
+import {HStack} from '@workday/canvas-kit-react/layout';
+import {plusIcon, caretDownIcon} from '@workday/canvas-system-icons-web';
+import {space, colors} from '@workday/canvas-kit-react/tokens';
+import styled from '@emotion/styled';
+
+const getDropdownColors = () => {
+  return {
+    default: {
+      background: colors.berrySmoothie400,
+      icon: colors.frenchVanilla100,
+      label: colors.frenchVanilla100,
+    },
+    hover: {
+      background: colors.berrySmoothie500,
+      label: colors.frenchVanilla100,
+    },
+    active: {},
+    focus: {},
+    disabled: {},
+  };
+};
+
+const StyledPrimaryButton = styled(PrimaryButton)({
+  height: space.l,
+  padding: space.xxs,
+});
+
+export const CustomStyles = () => (
+  <HStack spacing="s" padding="s">
+    <PrimaryButton
+      padding={space.l}
+      border={'`1px dotted red'}
```

Comment:
i think you need to remove the ` here. When I got to the example I do not see this border

Reply:
oo good catch, thanks!